enterprise/server/githubauth/githubauth.go

```diff
@@ -211,28 +220,28 @@ func (a *githubAuthenticator) SSOEnabled() bool {
 	return false
 }
 
-func (a *githubAuthenticator) authenticateUser(w http.ResponseWriter, r *http.Request) (*claims.Claims, *github.GithubUserResponse, error) {
+func (a *githubAuthenticator) authenticateUser(w http.ResponseWriter, r *http.Request) (*claims.Claims, *github.GithubUserResponse, *tables.Session, error) {
```

Comment:
nit: it might be a bit cleaner to have a struct like

```
type authenticatedGitHubUser struct {
    Profile *github.GithubUserResponse
    AccessToken string
}
```

and then return that from this func / store it in ctx under `auth.githubuser`

Reply:
Much cleaner, done!